### PR TITLE
Migration of networking, playback, serveractivity, mediaLibrary to ES6 Modules

### PR DESCRIPTION
**Pull request by Camc314**
**Issues**
Part of #868

### CODE REVIEW

**Comment by dkanada** [comment]
Somehow three extra lines were added to the ES6 list, even though they were already merged in another branch. @MrTimscampi do you think those would cause issues if merged? No clue why they're shown if they already exist in the target branch.

**Comment by MrTimscampi** [comment]
> Somehow three extra lines were added to the ES6 list, even though they were already merged in another branch. @MrTimscampi do you think those would cause issues if merged? No clue why they're shown if they already exist in the target branch.

I'm not sure what the behavior would be. If it passes the build CI step, it should be fine.

File: src/controllers/dashboard/networking.js [hunk 1]
```diff
@@ -136,7 +141,7 @@ define(['loading', 'libraryMenu', 'globalize', 'emby-checkbox', 'emby-select'],
         });
         view.querySelector('#btnSelectCertPath').addEventListener('click', function () {
             require(['directorybrowser'], function (directoryBrowser) {
-                var picker = new directoryBrowser.default();
+                const picker = new directoryBrowser();
```
Comment (dkanada) [inline src/controllers/dashboard/networking.js:144]: Just curious, was `.default()` changed in another pull request or was it never required here?

Comment (Camc314) [reply src/controllers/dashboard/networking.js:144]: Thanks for noticing that, I forgot to remove a require statement above. I think I was there in a previous PR? I'm not too sure. Either way, searching for a SSL cetificate is currently broken so I'll make a new PR to fix it

**Pull request merged**